tests/pr_test.go

```diff
@@ -139,6 +140,74 @@ func TestFullyConfigurable(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 }
 
+func TestQuickstartDefaultConfigSchematics(t *testing.T) {
+	t.Parallel()
+
+	options := testschematic.TestSchematicOptionsDefault(&testschematic.TestSchematicOptions{
+		Testing: t,
+		Region:  "eu-de",
```

Comment:
Any specific reason that you hard coded the region in these new tests? Can't we choose randomly?